src/app/layout/header/header.component.ts

```diff
@@ -5,67 +5,39 @@ import { Observable, Subscription } from 'rxjs';
 
 import { Broadcaster, Logger } from 'ngx-base';
 import { BsModalRef, BsModalService } from 'ngx-bootstrap/modal';
-import { Context, Contexts } from 'ngx-fabric8-wit';
+import { Context, Contexts, Space, Spaces, SpaceService } from 'ngx-fabric8-wit';
 import { AuthenticationService, User, UserService } from 'ngx-login-client';
+import { ContextLink, HeaderService, MenuItem, SystemStatus } from 'osio-ngx-framework';
 
 import { Navigation } from '../../models/navigation';
 import { DummyService } from '../../shared/dummy.service';
 import { LoginService } from '../../shared/login.service';
-import { MenuedContextType } from './menued-context-type';
 
+/*
 interface MenuHiddenCallback {
   (headerComponent: HeaderComponent): Observable<boolean>;
 }
+*/
```

Comment:
May I suggest to either remove this block as a whole or use `//` for commenting out multiline code?

Reply:
The block will be removed when this is complete.